tests/testthat/test-post-tweet.R

```diff
@@ -29,3 +29,27 @@ test_that("post_tweet works", {
   rt <- expect_message(post_destroy(crt$id_str), "your tweet has been deleted!")
   expect_equal(httr::status_code(rt), 200L)
 })
+
+test_that("post_tweet geolocated works", {
+  # Test errors
+  msg <- paste("test geolocated error", Sys.time()) # To avoid having duplicated status
+  expect_error(post_tweet(msg, lat = "x", long = 0))
```

Comment:
It's better to use `expect_snapshot()` for this sort of test because (a) that ensures you're generating the correct error and (b) it's allow the reviewer to see the exact text of the generated error.

Reply:
Hi @hadley , thanks for your feedback. Should I send a new PR, as this has been already merged?

 I would need some time as I am not familiar with `expect_snapshot` and friends.

Regards

Reply:
Yes, that's a good idea.

Reply:
See #541 , I would appreciate further guidance if needed. Regards